text/0000-channel-participation-api-without-system-channel.md

```diff
@@ -0,0 +1,497 @@
+from one of the other OSNs.
+
+If the channel already exists, the call will fail.
+
+If the system channel exists, the call will fail. 
+          
+The OSN receiving this call will inspect the block and conduct some sanity checks in order to make sure that the block 
+is well-formed.
+
+The OSN will try to find itself in the consenters set, and determine if it is a follower or a member of the cluster
+executing consensus. 
+
+If it is not in the set, it will continue as a **follower**, and start to pull blocks from other OSNs. It will continue
+to do so until it catches up with the config block received in the invocation. From then on, it will continue to pull blocks, 
+but will inspect each incoming config block and check whether it is included in the consenters set. If it is, it will 
+start the runtime components that execute consensus, thus joining the cluster. 
+
+If the OSN is in the consenters set, it will continue as **member**, first pulling blocks from other OSMs util it catches
+up with the given config block, and then starting the runtime components that execute consensus, thus joining the cluster.
+
+Note that joining an OSN as a member means that the channel was updated to include that OSN in the consenters set before it
+was joined by the participation API. Therefore, during the catch-up process and until the new OSN starts executing 
+consensus, the cluster is a state of reduced fault tolerance. For long chains this might be a problem, since catch-up
+can take a long time. The solution is to join the new OSN as a follower. After it catches up with the cluster the 
+channel admin can update the channel config and include it in the consenters set.
+
+The new OSN can pull blocks from other OSNs only if it has authorization to do so. A minimal requirement is that its 
+owning org will be defined in the channel configuration, and that the signature of the new OSN on the `Deliver` 
+request satisfy the `/Channel/Readers` policy.
+
+#### Creating a channel genesis block
+The way to create a channel genesis block is by using the `configtxgen`  tool, which already supports the 
+generation of an **application** channel genesis block.
+
+```configtxgen -outputBlock channel_genesis_block.pb -profile SampleRaftChannelCreation -channelID my-app-channel```
+
+Generating the channel genesis block follows the same rationale as generating the system channel genesis block, when 
+bootstrapping a new Fabric network. 
+
+The command refers to a profile `SampleRaftChannelCreation` in a `configtx.yaml` file, as usual. The YAML file 
+will point to the file-system path of the MSP configuration of each org, for application as well as ordering orgs. 
+In addition, it will point to the path of the certificates of the consenters. Generating a genesis block for an 
+application channel means that:
+- the `configtx.yaml` **does not** include a `Consortiums` config group
+- the `configtx.yaml` **must include** a `Channel/Application` config group
```

Comment:
`configtx.yaml` should not be explicitly referenced here. It's a (poor) means to an end.

Reply:
Done, I demoted that to an example.